docs/units/equivalencies.rst

```diff
@@ -142,23 +142,26 @@ These three conventions are implemented in
     >>> (116e9 * u.Hz).to(u.km / u.s, equivalencies=freq_to_vel)  # doctest: +FLOAT_CMP
     <Quantity -1895.4321928669085 km / s>
 
-Spectral Flux Density Units
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Spectral Flux / Luminosity Density Units
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-There is also support for spectral flux density units. Their use is
-more complex, since it is necessary to also supply the location in the
-spectrum for which the conversions will be done, and the units of
-those spectral locations.  The function that handles these unit
-conversions is :func:`~astropy.units.equivalencies.spectral_density`. This
-function takes as its arguments the |quantity| for the spectral
-location. For example::
+There is also support for spectral flux and luminosity density units. Their use
+is more complex, since it is necessary to also supply the location in the
+spectrum for which the conversions will be done, and the units of those spectral
+locations.  The function that handles these unit conversions is
+:func:`~astropy.units.equivalencies.spectral_density`. This function takes as
+its arguments the |quantity| for the spectral location. For example::
 
     >>> (1.5 * u.Jy).to(u.erg / u.cm**2 / u.s / u.Hz,
```

Comment:
I wonder why this example is there, as it does not need the equivalency. Maybe replace it with a photon one?